includes/class-shortcode.php

```diff
@@ -4,30 +4,37 @@ class Mad_Mimi_Shortcode {
 
 	public function render( $atts ) {
 
-		extract( shortcode_atts( array(
+		$atts = shortcode_atts( array(
 			'id' => false,
-		), $atts ) );
+		), $atts );
+
+		if ( ! $atts['id'] ) {
```

Comment:
Oh I see that is a default - however it is _possible_ someone could find a `shortcode_atts` filter and remove that default, which would result in an error.